### ISSUE TITLE
Follow up of #33780 - The metrics `Brave.Rewards.CustomNotificationPosition` isn't shown when moved the notifications to Top left, Top right and Bottom center

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#36725]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/33780
Follow up of #33780 - The metrics `Brave.Rewards.CustomNotificationPosition` isn't shown when moved the notifications to Top left, Top right and Bottom center

## Steps to Reproduce

   1. Verified the test plan from https://github.com/brave/brave-core/pull/20620
 

## Actual result:

Please see https://github.com/brave/brave-browser/issues/33780#issuecomment-1973065604

![image](https://github.com/brave/brave-browser/assets/38657976/a00917c4-ab2e-4f21-afe5-62ecf3cb3c73)

## Expected result:
The metrics `Brave.Rewards.CustomNotificationPosition` should be shown with response value

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.64.102 Chromium: 122.0.6261.111 (Official Build) beta (64-bit)
-- | --
Revision | c9343c77bd5dee3524fc45f89721ce0bf4c09b8d
OS | Windows 10 Version 22H2 (Build 19045.4046)

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @DJAndries @tmancey @mattmcalister @btlechowski

**Comment by aseren** [comment]
@GeetaSarvadnya @DJAndries, the name of the metric is `Brave.Rewards.CustomNotificationAdPosition`. It is set when user moves the custom notification ad.

**Comment by kjozwiak** [comment]
@DJAndries @tmancey I labelled the above as `Done`. Does the wiki require updating? The above requires `1.65.94` or higher for `1.65.x` verification 👍

**Comment by DJAndries** [comment]
> I labelled the above as Done. Does the wiki require updating?

no new metric was introduced with this issue/PR, so no need for a wiki update :+1:

**Comment by GeetaSarvadnya** [comment]
Verification INPROGRESS on

```
Brave | 1.65.89 Chromium: 123.0.6312.58 (Official Build) beta (64-bit)
-- | --
Revision | 2975bd8d3905ed79de82dd9fa11b6e4019075dd6
OS | Windows 10 Version 22H2 (Build 19045.4170)
```

**Comment by GeetaSarvadnya** [comment]
@kjozwiak: I am  verifying the issue on `1.65.95` I don't see the fix available in Beta. Is this merged in 1.65.95?
